tests/integration/targets/ntnx_objects_v2/tasks/deploy_object_store.yml

```diff
@@ -0,0 +1,359 @@
+        using the certificate details file:/tmp/certificate_details.json"
+    success_msg: "Create certificate for an object store using check mode passed"
+    fail_msg: "Create certificate for an object store using check mode failed"
+
+- name: Create certificate for an object store
+  nutanix.ncp.ntnx_object_stores_certificate_v2:
+    object_store_ext_id: "{{ object_store_ext_id }}"
+    path: "/tmp/certificate_details.json"
+  register: result
+  ignore_errors: true
+
+- name: Create certificate for an object store status
+  ansible.builtin.assert:
+    that:
+      - result.changed is true
+      - result.failed is false
+      - result.response is defined
+      - result.task_ext_id is defined
+      - result.response.status == "SUCCEEDED"
+    success_msg: "Certificate is created successfully"
+    fail_msg: "Certificate is not created successfully"
+
+- name: List all certificates for an object store
+  nutanix.ncp.ntnx_object_stores_certificate_info_v2:
+    object_store_ext_id: "{{ object_store_ext_id }}"
+  register: result
+  ignore_errors: true
+
+- name: List all certificates for an object store status
+  ansible.builtin.assert:
+    that:
+      - result.changed is false
+      - result.failed is false
+      - result.response is defined
+      - result.response | length == 1
+      - result.response[0].ext_id != "{{ certificate_ext_id }}"
+    success_msg: "All certificates are listed successfully"
+    fail_msg: "All certificates are not listed successfully"
+
+- name: Set certificate external ID
+  ansible.builtin.set_fact:
+    certificate_ext_id: "{{ result.response[0].ext_id }}"
+
+- name: Fetch certificate details using external ID
+  nutanix.ncp.ntnx_object_stores_certificate_info_v2:
+    object_store_ext_id: "{{ object_store_ext_id }}"
+    ext_id: "{{ certificate_ext_id }}"
+  register: result
+
+- name: Fetch certificate details using external ID status
+  ansible.builtin.assert:
+    that:
+      - result.changed is false
+      - result.failed is false
+      - result.response is defined
+      - result.response.ext_id == "{{ certificate_ext_id }}"
+    success_msg: "Certificate details are fetched successfully using external ID"
+    fail_msg: "Certificate details are not fetched successfully using external ID"
+
+- name: List all object stores
+  nutanix.ncp.ntnx_object_stores_info_v2:
+  register: result
+
+- name: List all object stores status
+  ansible.builtin.assert:
+    that:
+      - result.changed is false
+      - result.failed is false
+      - result.response is defined
+      - result.response | length > 0
+    success_msg: "All object stores are listed successfully"
+    fail_msg: "All object stores are not listed successfully"
+
+- name: Set object store name
+  ansible.builtin.set_fact:
+    object_store_name: "{{ result.response[0].name }}"
+
+- name: List all object stores with filter
+  nutanix.ncp.ntnx_object_stores_info_v2:
```

Comment:
We need to have more than 1 object store in setup. Then only this test is verified. What if incase setup have 1 object store and internally filter doesn't work then it will show that one object store, which will pass the test which it shoudln't